osu.Game/Tests/Visual/RateAdjustedBeatmapTestScene.cs

```diff
@@ -12,8 +12,11 @@ protected override void Update()
         {
             base.Update();
 
-            // note that this will override any mod rate application
-            Beatmap.Value.Track.Tempo.Value = Clock.Rate;
+            if (Beatmap.Value.TrackLoaded && Beatmap.Value.Track != null) // null check... wasn't required until now?
```

Comment:
Forgotten about comment? Not sure about this - you'd would hope track is non-null (at the very minimum it should be a virtual track), so something looks wrong.

Reply:
Looks to have since been resolved, have reverted.